migration/fluentd-wikigridd-to-bitnami.md

```diff
@@ -0,0 +1,15 @@
+# Upgrade to fluentd chart from wikigrid to bitnami
+
+Due to our current nginx ingress setup we can only have one deployment running at a time which means that this upgrade will introduce some downtime.
```

Comment:
nginx-ingress?

Reply:
> nginx-ingress?

That must be  a mix-up as I copied the nginx-ingress migration file :)